sgkit_vcf/tests/test_utils.py

```diff
@@ -0,0 +1,20 @@
+from pathlib import Path
+
+from sgkit_vcf.utils import temporary_directory
+
+
+def test_temporary_directory():
+
+    prefix = "prefix-"
+    suffix = "-suffix"
+    with temporary_directory(suffix=suffix, prefix=prefix) as tmpdir:
+
+        dir = Path(tmpdir)
+        assert dir.exists()
+        assert Path(tmpdir).name.startswith(prefix)
```

Comment:
Duplicating ``Path(tmpdir)`` here

Reply:
Fixed in https://github.com/pystatgen/sgkit-vcf/pull/9/commits/d7ea0012c58ab58410e692b24e89778c769395dd